### PR TITLE
Task mailer

### PR DESCRIPTION
**Pull request by atd**

### CODE REVIEW

File: config/initializers/smtp.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# I can't belive smtp:// urls are not suported by Rails
```
Comment (nogates) [inline config/initializers/smtp.rb:1]: :scream:

File: spec/mailers/task_mailer_spec.rb [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+    let(:task)  { tasks(:weekly) }
+
+    it "should build the email" do
+      email = TaskMailer.notification(task)
+
+      expect(email.to).to match_array(task.turns.first.emails)
+      expect(email.cc).to include(task.email)
+      expect(email.subject).to include(task.turns.first.groups.first.name)
+      expect(email.text_part.body.to_s).to include(task.turns.first.groups.first.name)
```
Comment (nogates) [inline spec/mailers/task_mailer_spec.rb]: IMHO, it's better to separete different expectations on differents examples. That way, it's easier to debug when you got a fail (you can jump to the example line), and you know exactly which test has failed without even looking at it. May I suggest this:

```ruby
describe "when task has turns" do
  let(:task)  { tasks(:weekly) }
  let(:email) { TaskMailer.notification(task) }

  it 'the email has the valid TO' do
    expect(email.to).to match_array(task.turns.first.emails)
  end

  it 'the email has the valid CC' do
    expect(email.cc).to include(task.email)
  end

  it 'the email has the valid subject' do
    expect(email.subject).to include(task.turns.first.groups.first.name)
  end

  it 'the email has the valid text_part' do
    expect(email.text_part.body.to_s)
      .to include(task.turns.first.groups.first.name)
  end
end
```

**Review by nogates — APPROVED** [review]
LGTM One picky comment about rspec standards

File: spec/mailers/task_mailer_spec.rb (hunk 2 above)
Comment (atd) [reply spec/mailers/task_mailer_spec.rb]: Sure! I just adapted the example from http://guides.rubyonrails.org/testing.html#testing-your-mailers using minitest

I am changing it

**Pull request merged**